src/docs/docsite.json

```diff
@@ -115,64 +115,82 @@
   "tree": [
     { "page": "index",
       "children": [
-        { "heading": "Getting Started" },
-        { "page": "install" },
-        { "page": "setup_repo" },
-        { "page": "first_tutorial" },
-        { "page": "common_tasks" },
 
-        { "heading": "Pants Basics"},
-        { "page": "why_use_pants" },
-        { "page": "first_concepts" },
-        { "page": "build_files" },
-        { "page": "target_addresses" },
-        { "page": "3rdparty" },
-        { "page": "options"},
-        { "page": "invoking" },
-        { "page": "reporting_server" },
-        { "page": "ide_support" },
+       {"heading": "Getting Started"},
+       {"pages": [
+                     "install", 
+                     "setup_repo",
+                     "first_tutorial",
+                     "common_tasks"
+                ]
+            
+       },
+       {"heading": "Pants Basics"},
+       {"pages" : [
+                    "why_use_pants",
+                    "first_concepts",
+                    "build_files",
+                    "target_addresses",
+                    "3rdparty",
+                    "options",
+                    "invoking",
+                    "reporting_server",
+                    "ide_support"
+                ]
+       },
+       {"collapsible_heading": "JVM Support",
+        "pages" : [
+                    "jvm_projects",
+                    "3rdparty_jvm",
+                    "scala",
+                    "publish",
+                    "from_maven"
+                ]
+       },
+       {"collapsible_heading": "Python Support",
+        "pages" : [
+                    "python_readme",
+                    "3rdparty_py"
+                ]
+       },
+       {"pages" : ["go_readme"]},
+       {"collapsible_heading": "Code & Doc Gen"},
+       {"pages" : [
```

Comment:
I think these two js objects should be made into one, like the other collapsibles.